script/Arcueid/Day 4/04_00_ARC04_1.txt

```diff
@@ -127,17 +132,18 @@ Looking closely, I notice my school uniform in her hand.
 [sha:1220f042662d99c688840403e03c6666edd1d607]{
 -- Page 12, Offset 5594.
 -- 「……はい。重ねて申し訳ありません。
-"Yes. Once again, I apologize.
+"Yes. Once again, I must apologize.
 }
 [sha:b3d651a3c00432bbc88c78c2d0187e8c2fb25d63]{
 -- Page 12, Offset 5595.
 -- お見苦しいところをお見せしました」
-For making you see something so unpleasant."
+I caused you to see something unpleasant."
```

Comment:
oh cmon
>It seems I have embarrassed myself in front of you.

something like this. this phrase is uttered when you commit a faux pas or whatever

5598
>I don't think she did anything particularly embarrassing, but...